vlm/src/main/scala/WarpRasterSource.scala

```diff
@@ -0,0 +1,58 @@
+package geotrellis.contrib.vlm
+
+import geotrellis.proj4._
+import geotrellis.raster._
+import geotrellis.raster.reproject.{ReprojectRasterExtent, Reproject}
+import geotrellis.vector._
+
+import java.net.URI
+
+
+class WarpRasterSource(
```

Comment:
This definitely needs a spec, probably something similar to `TileRDDReprojectSpec` where you compare reprojecting a whole raster that was produced using `WarpRasterSource`. It would probably be good idea to read 4 windows that divide the target raster and stitch them together to make sure that resampling is happening correctly around the tile borders.